what-the-stack/src/main/java/com/haroldadmin/whatthestack/WhatTheStackActivity.kt

```diff
@@ -28,7 +26,8 @@ class WhatTheStackActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_what_the_stack)
+        val activityWhatTheStackBinding = ActivityWhatTheStackBinding.inflate(layoutInflater)
```

Comment:
Please follow the naming convention for binding variables here too.

Reply:
done